test/specs/behaviors/testHelper.tsx

```diff
@@ -96,4 +117,20 @@ export class TestHelper {
     }
     return stringToConvert
   }
+
+  private verifySpecificationTag(behaviorFileName: string) {
+    if (!excludedFiles.find(item => item === behaviorFileName)) {
+      test(`${behaviorFileName} : File is missing specification tag.`, () => {
+        fail(
+          `File should have specification tag. If tests are written in separate file then add behavior file name into 'excludedFiles'.`,
```

Comment:
actually, it is not `file` but `accessibility behavior` that should have a `specification` tag :)